yaptide/routes/celery_routes.py

```diff
@@ -165,14 +166,20 @@ def delete(user: UserModel):
 
         # The merge_id is canceled first because merge task starts after run simulation tasks are finished/canceled.
         # We don't want it to run accidentally.
-        celery_app.control.revoke(simulation.merge_id, terminate=True, signal="SIGINT")
-        celery_app.control.revoke(celery_ids, terminate=True, signal="SIGINT")
-        update_simulation_state(simulation=simulation, update_dict={"job_state": EntityState.CANCELED.value})
-        for task in tasks:
-            if task.task_state in [EntityState.PENDING.value, EntityState.RUNNING.value]:
-                update_task_state(task=task, update_dict={"task_state": EntityState.CANCELED.value})
-
-        terminate_unfinished_tasks.delay(simulation_id=simulation.id)
+        if simulation.sim_type == "shieldhit" and simulation.job_state in (EntityState.RUNNING.value):
+            logging.debug('Inside cancelling shieldhit')
+            for task in tasks:
```

Comment:
`man kill` says you can kill multiple pid's at once:

![Uploading image.png…]()